blueprints/helper-addon/index.js

```diff
@@ -0,0 +1,3 @@
+/*jshint node:true*/
+
+module.exports = require('ember-cli/blueprints/addon-import');
```

Comment:
I think we should try to decouple. ember bundling ember-cli is no-beuno.

I admittedly though haven't investigated this deeply today, it is possible im missing something